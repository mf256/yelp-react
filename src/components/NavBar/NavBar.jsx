import React from 'react';
import styled from 'styled-components';
import { mainColor } from '../GlobalStyles/GlobalStyles';

const StyledNavBar = styled.div`
  background-color: ${mainColor};
  padding: 5px;
  text-align: center;
  width: 100%;
`;

const StyledHeading = styled.h1`
  margin: 0;
  color: #ffffff;
`;

class NavBar extends React.PureComponent {
  render() {
    return (
      <StyledNavBar>
        <StyledHeading>
          Yelp in React
        </StyledHeading>
      </StyledNavBar>
    );
  }
}

export default NavBar;
