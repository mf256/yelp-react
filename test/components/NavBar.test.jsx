import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../../src/components/NavBar/NavBar';

describe('NavBar', () => {
  it('should match snapshot', () => {
    const component = shallow(<NavBar />);

    expect(component).toMatchSnapshot();
  });
});
