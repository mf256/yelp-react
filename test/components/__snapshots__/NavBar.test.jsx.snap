// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<styled.div>
  <styled.h1>
    Yelp in React
  </styled.h1>
</styled.div>
`;
